doc/release/0.18.0-notes.rst

```diff
@@ -65,18 +70,124 @@ zero-phase low-pass FIR filter, and downsamples using `scipy.signal.upfirdn`.
 This method can be faster than FFT-based filtering provided by
 `scipy.signal.resample` for some signals.
 
-`scipy.sparse` compatibility improvements
+`scipy.signal.firls` constructs FIR filters using least-squares error minimization.
+
+
+Discrete-time linear systems
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`scipy.signal.dlti` provides an implementation of discrete-time linear systems.
+Accordingly, the `StateSpace`, `TransferFunction` and `ZerosPolesGain` classes
+have learned a the new keyword, `dt`, which can be used to create discrete-time
+instances of the corresponding system representation.
+
+
+`scipy.sparse` improvements
 -----------------------------------------
 
 The functions `sum`, `max`, `mean`, `min`, `transpose`, and `reshape` in
 `scipy.sparse` have had their signatures augmented with additional arguments
 and functionality so as to improve compatibility with analogously defined
 functions in `numpy`.
 
+Sparse matrices now have a `count_nonzero` method, which counts the number of
+nonzero elements in the matrix. Unlike `getnnz()` and ``nnz`` propety,
+which return the number of stored entries (the length of the data attribute),
+this method counts the actual number of non-zero entries in data.
+
+
+`scipy.optimize` improvements
+-----------------------------
+
+The implementation of Nelder-Mead minimization,
+`scipy.minimize(..., method="Nelder-Mead")`, obtained a new keyword,
+`initial_simplex`, which can be used to specify the initial simplex for the
+optimization process.
+
+Initial step size selection in CG and BFGS minimizers has been improved. We
+expect that this change will improve numeric stability of optimization in the
+vast majortiy of use cases. See pull request #5536 for details.
```

Comment:
and isn't "vast majority" a bit of an overstatement? numerical stability is already fine for most usages right?